flexidate/__init__.py

```diff
@@ -32,7 +31,7 @@ def __init__(self, year=None, month=None, day=None, qualifier=''):
          
     def _cvt(self, val, rjust=2, force=False):
         if val:
-            tmp = unicode(val).strip()
+            tmp = str(val).strip()
```

Comment:
Is this backwards compatible with python 2? I'm not sure tests would catch this so we may need to add a test with unicode stuff ...